tests/VisualStudio.Roslyn.SDK/Roslyn.SDK.IntegrationTests/InProcess/InProcComponent.cs

```diff
@@ -30,25 +30,26 @@ protected async Task ExecuteCommandAsync(string commandName, string args = "")
         {
             await JoinableTaskFactory.SwitchToMainThreadAsync();
 
-            var dte = await GetGlobalServiceAsync<SDTE, EnvDTE.DTE>();
+            var dte = await GetRequiredGlobalServiceAsync<SDTE, EnvDTE.DTE>();
             dte.ExecuteCommand(commandName, args);
         }
 
-        protected async Task<TInterface> GetGlobalServiceAsync<TService, TInterface>()
+        protected async Task<TInterface> GetRequiredGlobalServiceAsync<TService, TInterface>()
```

Comment:
💭 What if we keep the same name but have the new behavior?

Reply:
I am find reverting it. I typically use `Required` for things that throw but considering the usage I am also ok with reverting it.